packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -86,4 +114,15 @@ export class OpGroupingManager {
 			compression: subMessage.compression,
 		}));
 	}
+
+	public shouldGroup(batch: IBatch): boolean {
+		return (
+			// Grouped batching must be enabled
+			this.config.groupedBatchingEnabled &&
+			// The number of ops in the batch must surpass the configured threshold
+			batch.content.length >= this.config.opCountThreshold &&
+			// Support for reentrant batches must be explicitly enabled
+			(this.config.reentrantBatchGroupingEnabled || batch.hasReentrantOps !== true)
```

Comment:
Sharing some observations:
1. I'd remove this.config.groupedBatchingEnabled gate. You can control it fully with this.config.opCountThreshold gate, if it has much higher (infinite?) default value.
2. This would also allow you to control it only for new clients, not caring about screwing old clients, who do not know anything about opCountThreshold or reentrantBatchGroupingEnabled 
3. Is the intention here to go with this flow:
   - first, go with something like {opCountThreshold : 2, reentrantBatchGroupingEnabled : false } and validate this works fine.
   - Then go with {opCountThreshold : 2000, reentrantBatchGroupingEnabled : true} or so and validate we are Ok
   - Move to 100% enabled.

This looks fine, but it would be great for first two steps to be enabled at the same time (i.e. not take away). But I think we could achieve it simply by shipping code change and changing logic - it does not matter really if it's FG change or not.

Reply:
> I'd remove this.config.groupedBatchingEnabled gate.

This is not a gate, it's coming from a runtime config. This feature is enabled by Loop, via config, based on a feature gate in the Loop code. We don't have a feature gate to enable it on the Fluid side.